locale/circuitpython.pot

```diff
@@ -434,6 +434,12 @@ msgstr ""
 msgid ", in %q\n"
 msgstr ""
 
+#: shared-bindings/displayio/Display.c
+#: shared-bindings/displayio/EPaperDisplay.c
+#: shared-bindings/framebufferio/FramebufferDisplay.c
```

Comment:
I think weblate will update this automatically. The other messages in these files haven't been updated either.